docker/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM fedora:26
+MAINTAINER mmarusak@redhat.com
+
+# Add copr repo
+ADD docker/files/faf.repo /etc/yum.repos.d/
+
+RUN useradd faf && \
```

Comment:
Should be: `useradd --system -g faf -d /etc/faf -s /sbin/nologin faf`

Reply:
useradd: group 'faf' does not exist
Are you sure about the `-g faf` part? 
From useradd man pages: "By default, a group will also be created for the new user"
You probably meant -U `Create a group with the same name as the user, and add the user to this group.`, haven't you?